### ISSUE TITLE
Animation bug

### ISSUE THREAD

**Issue by ikok07** [markiv/SwiftUI-Shimmer#19]
### When I launch my application for the first time the animation is not applied only to the redacted view but to the whole view:

**This is my code:** 
```swift
ScrollView {
                    TodayFeaturedRecipesView()
                        .environmentObject(viewModel)
                        .tag(0)
                    
                    TodayStatisticsView()
                        .environmentObject(viewModel)
                        .padding(.top)
                    
                    if !(accManager.user?.isPremium ?? false) {
                        TodayFreeRecipesView()
                            .environmentObject(viewModel)
                        ExplorePremiumBannerView()
                            .redacted(reason: viewModel.isLoading ? .placeholder : [])
                            .shimmering(active: viewModel.isLoading, gradient: colorScheme == .dark ? K.Gradients.lightShimmeringGradient : K.Gradients.darkShimmeringGradient)
                    }
                }
```

![image](https://github.com/markiv/SwiftUI-Shimmer/assets/70591797/25631f18-e661-4e30-bdad-2594f061e973)

![image](https://github.com/markiv/SwiftUI-Shimmer/assets/70591797/3867c8f1-98ac-41da-9430-8c61243a39ed)

**Comment by ikok07** [comment]
### This is my code: 
```swift
ScrollView {
                    TodayFeaturedRecipesView()
                        .environmentObject(viewModel)
                        .tag(0)
                    
                    TodayStatisticsView()
                        .environmentObject(viewModel)
                        .padding(.top)
                    
                    if !(accManager.user?.isPremium ?? false) {
                        TodayFreeRecipesView()
                            .environmentObject(viewModel)
                        ExplorePremiumBannerView()
                            .redacted(reason: viewModel.isLoading ? .placeholder : [])
                            .shimmering(active: viewModel.isLoading, gradient: colorScheme == .dark ? K.Gradients.lightShimmeringGradient : K.Gradients.darkShimmeringGradient)
                    }
                }
```

**Comment by ast3150** [comment]
We had the same issue and were able to fix it by introducing the changes from #20 

Can you try if this fix works for you as well?

**Comment by ikok07** [comment]
Yup, the issue was fixed! Thank you :)
